.travis.yml

```diff
@@ -5,6 +5,9 @@ php:
   - '5.5'
   - '5.6'
   - '7.0'
+  - '7.1'
+  - 'hhvm'
```

Comment:
We fail on `hhvm`. Unclear why, just going to disable it for now and note in the README that we don't support `hiphop`.